src/entities/drawer/ui/DrawingBoard.tsx

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { FC, useEffect, useRef, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import {
+  Skia,
+  SkiaView,
+  SkPath,
+  TouchInfo,
+  PaintStyle,
+  useTouchHandler,
+  useDrawCallback,
+  SkCanvas,
+  Canvas,
+  Group,
+  Path,
+} from '@shopify/react-native-skia';
+
+import { colors } from '@app/shared/lib';
+import { Box } from '@app/shared/ui';
+
+import {
+  convertToSkPaths,
+  DrawLine,
+  DrawPoint,
+  Point,
+  transformByWidth,
+  transformBack,
+  ResponseSerializer,
+  DrawResult,
+} from '../lib';
+
+const paint = Skia.Paint();
+paint.setColor(Skia.Color(colors.black));
+paint.setStrokeWidth(1);
+paint.setStyle(PaintStyle.Stroke);
+
+type Props = {
+  initialLines: Array<DrawLine> | null;
+  onStarted: () => void;
+  onResult: (result: DrawResult) => void;
+  width: number;
+};
+
+const DrawingBoard: FC<Props> = props => {
+  const currentPathRef = useRef<SkPath | null>();
+
+  const canvasRef = useRef<SkCanvas>();
+
+  const logLines = useRef<Array<DrawLine>>([]).current;
+
+  const [paths, setPaths] = useState<Array<SkPath>>([]);
+
+  const { initialLines, onResult, onStarted, width } = props;
+
+  useEffect(() => {
+    if (!width || !initialLines || !initialLines.length) {
```

Comment:
`!initialLines || !initialLines.length`

you can shorten it to `!initialLines?.length`